File: 3005-count-elements-with-maximum-frequency/3005-count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        map<int,int>mpp;
        int mx=0,ans=0;
        for(auto &it:nums){
            mpp[it]++;
        }
        for(auto &it:mpp){
            if(it.second>mx){
                mx=it.second;
                ans=it.second;
            }else if(it.second==mx){
                ans+=it.second;
            }
        }
        return ans;
    }
};

File: 3005-count-elements-with-maximum-frequency/3005-count-elements-with-maximum-frequency.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxFrequencyElements = function(nums) {
    const mpp=new Map();
    nums.forEach((i)=>{
        mpp.set(i,(mpp.get(i)||0)+1);
    });
    let mx=Math.max(...mpp.values());
    let arr=[...mpp.keys()].filter(num=> mpp.get(num)===mx);
    // console.log(mx,arr)
    return mx*arr.length;
    
};